src/smart-components/portfolio/portfolio-item-detail/portfolio-item-detail.js

```diff
@@ -106,17 +106,21 @@ const PortfolioItemDetail = () => {
 
   const availability =
     portfolioItemData?.source?.availability_status || 'unavailable';
-  const unavailable = [portfolioItemData?.source]
-    .filter(({ notFound }) => notFound)
-    .map(({ object }) => (
-      <Alert
-        className="pf-u-mb-sm"
-        key={object}
-        variant="warning"
-        isInline
-        title={formatMessage(portfolioMessages.objectUnavaiable, { object })}
-      />
-    ));
+  const unavailable = portfolioItemData?.source
+    ? [portfolioItemData?.source]
+        .filter(({ notFound }) => notFound)
+        .map(({ object }) => (
+          <Alert
+            className="pf-u-mb-sm"
+            key={object}
+            variant="warning"
+            isInline
+            title={formatMessage(portfolioMessages.objectUnavaiable, {
+              object
+            })}
+          />
+        ))
+    : [];
```

Comment:
It's possible I'm just really bad at reading javascript, but it feels very weird to me to see 10+ lines and a ternary operator. I'm not sure what the "javascript" way of cleaning this up would be, but maybe taking lines 110 through 122 into a separate function and simply calling that would make this easier to grok. It would probably all fit on line 109 then, as well.
```
const unavailable = portfolioItemData?.source ? <somethingWithAGoodName(portfolioItemData)> : []
```